lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -218,10 +271,11 @@ def configure(conf)
         begin
           @annotations_regexps << Regexp.compile(regexp)
         rescue RegexpError => e
-          log.error "Error: invalid regular expression in annotation_match: #{e}"
+          log.warn "Error: invalid regular expression in annotation_match: #{e}"
```

Comment:
Why change this from `error` to `warn`?  e.g. this is something I would want to know about asap if I typed in an incorrect regexp.